static/styles/left_sidebar.css

```diff
@@ -465,6 +465,21 @@ li.expanded_private_message {
         position: relative;
         top: 0;
     }
+
+    .group_size {
```

Comment:
Maybe worth adding a comment that this is a near duplicate of the `unread_count` class?

Reply:
Added a comment, PTAL.